.gitlab-ci.yml

```diff
@@ -0,0 +1,25 @@
+image: $REPO_URL/stage
```

Comment:
Since this is a public package, plz follow the template in 
https://github.com/polyswarm/polyswarm-transaction/blob/master/.gitlab-ci.yml

Reply:
You also need to add a bumpversion file like this
https://github.com/polyswarm/polyscore-pipeline/blob/master/.bumpversion.cfg

Reply:
Maybe this is a better reference since we are using pipenv here
https://github.com/polyswarm/polyscore-pipeline/blob/master/.gitlab-ci.yml

Reply:
Can I use the bumpversion config section that exists in `setup.cfg` instead of extracting it out into `.bumpversion.cfg`?

Reply:
As long as the `bumpversion` commands work, you can. It will be used in ci automatically, to the things there have to work.